README.md

```diff
@@ -85,7 +91,7 @@ import solidityPlugin from 'prettier-plugin-solidity/standalone';
 
 async function format(code) {
   return await prettier.format(code, {
-    parser: "solidity-parse",
+    parser: "slang",
```

Comment:
If `slang` is the user-facing parser name, I wonder what do you think about `slang-solidity` instead?

1. Is the parser name is "shared" between all prettier plugins, so it has to be unique, so it doesn't collide with another plugin/language?
2. I wonder if you are ever planning on supporting [stand-alone assembly files](https://docs.soliditylang.org/en/latest/yul.html#stand-alone-usage
) in the future, and therefore, you might want to introduce a `slang-yul` parser as well?

On the other hand, maybe this is overengineering/overthinking, and we can just leave it up to the user config, or the plugin can just auto-detect that from the file extension.
Just thinking out loud.

Reply:
`slang-solidity` is a good idea.

Adding support for `.yul` is very straight forward as we just serve 2 parsers `slang-solidity` that expects a `SourceUnit` and `slang-yul` that expects a `YulBlock`.

Any thoughts on this @fvictorio?

Reply:
I'd say let's open an issue about this and not include it here, since this change is massive as is.